internal/flags/invoice_type.go

```diff
@@ -0,0 +1,35 @@
+package flags
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var invoiceType string
+
+func AddInvoiceType(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&invoiceType, "type", "issued", "type of the invoice. Possible values: 'all', 'upcoming', 'issued'")
+	_ = cmd.RegisterFlagCompletionFunc("type", invoiceTypeFlagCompletion)
+
+}
+
+func InvoiceType() (string, error) {
+	if err := validateInvoiceType(invoiceType); err != nil {
+		return "", err
+	}
+	return invoiceType, nil
+}
+
+func validateInvoiceType(invoiceType string) error {
+	switch invoiceType {
+	case "issued", "all", "upcoming", "":
```

Comment:
Why are you accepting `""`? Do we need to replace it with `issued`"?

Reply:
just being super safe. happy to remove it